bin/fai-cd

```diff
@@ -188,10 +188,15 @@ create_squashfs_image() {
 # - - - - - - - - - - - - - - - - - - - - - - - - - -
 create_grub2_image() {
 
-    mkdir -p $tmp/boot
+	mkdir -p $tmp/boot
+	# Needs 'grub2-common' and 'fonts-dejavu-core' packages.
+	grub-mkfont -s 16 -o "$tmp/DejaVuSansMono.pf2" /usr/share/fonts/truetype/dejavu/DejaVuSansMono.ttf
```

Comment:
@dzatoah White-space problem here? (Tab vs. spaces)

Reply:
Yes, fai-cd uses spaces vs. tabs very inconsistently. This was my attempt at fixing this a bit.
I think it uses tabs (rendered with 8 spaces) normally and spaces 4 spaces wherever there is an if-statement.
![grafik](https://github.com/faiproject/fai/assets/16420080/f5179b35-1e4b-4630-b1b6-e1dff4c63fc5)